### PR TITLE
[Brent] small small text changes

### PR DESCRIPTION
**Pull request by dracos**
[skip changelog]
Fixes all items on https://3.basecamp.com/4020879/buckets/33769634/todolists/6623008745

### CODE REVIEW

File: templates/web/base/waste/bulky/intro.html [hunk 1]
```diff
@@ -9,15 +9,18 @@ <h3>Before you start your booking</h3>
     <li>Requesting a [% service_name %] collection usually takes approximately <strong>10 minutes</strong></li>
```
Comment (MorayMySoc) [inline templates/web/base/waste/bulky/intro.html:9]: Requesting a small items collection usually takes around 10 minutes

File: templates/web/base/waste/bulky/intro.html [hunk 2]
```diff
@@ -9,15 +9,18 @@ <h3>Before you start your booking</h3>
...
   [% IF c.cobrand.moniker != 'brent' %]
     <li>You can request up to <strong>[% c.cobrand.bulky_items_maximum | numwords %] items per collection</strong></li>
-  [% ELSE %]
-    <li>You can arrange to have any of the following small items collected from outside your home for free, including textiles, small electricals, household batteries, and paint.
-    <li>Householders will be able to book up to three categories per collection. This could be, for example; 10 x AA batteries, two x 5L paint, one x black sack of textiles.</li>
-    <li>To see your options for coffee pod recycling please visit
-        <a href="https://www.podback.org/recycle-checker">https://www.podback.org/recycle-checker</a>
   [% END %]
     <li>Please ensure you have read the <strong><a href="[% c.cobrand.call_hook('bulky_tandc_link') %]" target="_blank">[% service_name %] collection</a></strong> page on the council’s website</li>
```
Comment (MorayMySoc) [inline templates/web/base/waste/bulky/intro.html:13]: Please make sure you have read the <strong>small items collection</strong> page on the council’s website.

File: templates/web/base/waste/bulky/intro.html [hunk 3]
```diff
@@ -9,15 +9,18 @@ <h3>Before you start your booking</h3>
...
+    <li>You can arrange to have any of the following small items collected from outside your home for free: textiles, small electricals, household batteries, and paint tins.
+    <li>Householders will be able to book up to <strong>three</strong> categories per collection. For example, a collection could include 10 AA batteries, two x 5 litre paint tins, one black sack of textiles.
+    <li>To see your options for coffee pod recycling please visit
+        <a href="https://www.podback.org/recycle-checker" target="_blank">https://www.podback.org/recycle-checker</a>
+  [% END %]
+  [% IF c.cobrand.moniker != 'brent' %]
     <li>In order to help us with your collection, we will ask you to optionally add pictures of the items to be collected and the location</li>
-    <li>Before confirming your booking, check all the info provided is correct</li>
+  [% END %]
+    <li>Before confirming your booking, check all the information provided is correct</li>
```
Comment (MorayMySoc) [inline templates/web/base/waste/bulky/intro.html:23]: Before confirming your booking, you need to check all the information provided is correct.

File: templates/web/base/waste/bulky/intro.html (hunk 1 above)
Comment (dracos) [reply templates/web/base/waste/bulky/intro.html:9]: Will make this general change.

File: templates/web/base/waste/bulky/intro.html (hunk 3 above)
Comment (dracos) [reply templates/web/base/waste/bulky/intro.html:23]: And this

**Pull request merged**